packages/components/src/components/hds/form/divider/index.hbs

```diff
@@ -0,0 +1,6 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes></div>
```

Comment:
[question] why not use the existing `Hds::Separator` component here? (with `@spacing="0"`)